src/v2/Apps/Order/Components/SavedAddressItem.tsx

```diff
@@ -39,13 +39,32 @@ export const SavedAddressItem: React.FC<SavedAddressItemProps> = (
           nameAndAddressLine.map(
             (line: string, index: number) =>
               line && (
-                <Text
-                  style={{ textTransform: "capitalize" }}
-                  variant="text"
-                  key={index}
-                >
-                  {line}
-                </Text>
+                <Flex key={index} justifyContent="row" alignItems="center">
+                  <Text
+                    style={{ textTransform: "capitalize" }}
+                    textColor={index === 0 ? "black100" : "black60"}
+                    variant="text"
+                  >
+                    {line}
+                  </Text>
+                  {address?.isDefault && index === 0 && (
+                    <Banner
+                      ml={0.5}
+                      variant="defaultLight"
+                      px={0.5}
+                      py={0.2}
+                      borderRadius={2}
```

Comment:
@dzucconi - do we have patterns in place around border-radius? I haven't seen this around except for in the `BorderBox` component

Reply:
Not really; might be worth surfacing to `#design-system`

Reply:
BTW: apparently we've been trying to drop border radiuses everywhere

Reply:
Also: I didn't even notice this until just now: you don't want to be giving custom borders/padding etc to these components.

Reply:
Just looking at the comp now, this just isn't a `Banner` — maybe a `Pill`?

Reply:
@dzucconi thanks, I'll use Pill